debian/copyright

```diff
@@ -1,5 +1,38 @@
-Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: revpi-repo
```

Comment:
If there is no public repo, don't add those fields?

Reply:
Are you mean it is not (be able to) submitted to debian community?
btw, the revpi-repo is now (going to be) public on github

Reply:
It looks ugly to have the template fields in the file. If it is public, add the repo. If not remove the lines!